### PR TITLE
[MRG] Use ipympl instead of notebook for compat jupyter notebook/lab

### PR DESCRIPTION
**Pull request by glemaitre**

### CODE REVIEW

**Comment by glemaitre** [comment]
This required to update to matplotlib 2.0 and follow the installation guide:
https://github.com/matplotlib/jupyter-matplotlib#installation

The annoying line is actually the following bash command which cannot be done in the environment:
```bash
jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

File: requirements.txt [hunk 1]
```diff
@@ -5,5 +5,9 @@ Pillow>=2.9.0
 numpy>=1.9.2
 scipy>=0.15.1
 scikit-learn>=0.18
-matplotlib>=1.4.3
+matplotlib>=2.0
 pandas>=0.19
+jupyter
+jupyterlab
```
Comment (amueller) [inline requirements.txt:11]: why is this a requirement?

**Comment by amueller** [comment]
I've seen issues with installing widgets in the past. Not sure what the current status is. Maybe removing the animations is easier.

**Comment by amueller** [comment]
Unless you want to volunteer to upgrade the different node.js versions installed on the windows laptops people will bring ;)

**Comment by glemaitre** [comment]
Let s go inline ;). I modified the requirements for Travis in fact.                                                                                                                                                                         Sent from my phone - sorry to be brief and potential misspell.

**Comment by amueller** [comment]
I guess we don't suggest to the participants to use requirements.txt so it's fine to add stuff to it.
I guess you won't have time to work on updating the interactive plots today (because it's late)?

**Comment by glemaitre** [comment]
It will be tight: pydata meetup. But put any issue I will address it tomorrow.                                                                                                                                                                         Sent from my phone - sorry to be brief and potential misspell.

**Comment by amueller** [comment]
Thanks and have fun. No rush.

**Pull request closed without merging**